src/components/routes/Host/ListingForm/AsideContent.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+
+interface AsideContentInterface {
+  [name: string]: React.ReactNode;
+}
+
+const HomeType = () => (
+  <div>
+    <strong>Entire place</strong>
+    <p>The whole space is dedicated to guests. This may be a house, apartment, condo, loft, etc.</p>
+    <br />
+    <strong>Private room</strong>
+    <p>A single room within a place is dedicated to guests. While other areas could be shared, guests have their own private room for sleeping.</p>
+    <br />
+    <strong>Shared room</strong>
+    <p>Bedrooms or a common area that could be shared with others are dedicated to guests.</p>
+  </div>
+);
+
+const FullAddress = () => (
+  <div>
+    <p>Don't forget to add a zip code!</p>
+  </div>
+);
+
+const Title = () => (
+  <div>
+    <p>This is ListingName Info</p>
```

Comment:
I'm wondering if these should e more use-friendly since they're consumer facing?  Maybe something like `This is the listing title`.  Same for the ones below this one:

`This is the description for the listing.`
`This is the country the listing is located in`
`This is the listing's cover photo`
..etc

Reply:
oh these were just placeholders haha. The scaffolding was more of the goal of this PR, prob should have been more clear soz.

Reply:
updated match the listing doc: https://docs.google.com/document/d/1w1RCVl9lIHqGA5UOVk3B0zqgRG3qSDBlfsQP_iiF0hg/edit#

some fields don't have explanations -- left them blank for now.